language/move-prover/interpreter/crypto/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Shall we add a comment indicating the reason why we duplicated the signature verification logic? My understanding is that this will go away once we restructure the prover, and it might be helpful to make that intention clear.

Reply:
Comment added